packages/pds/src/index.ts

```diff
@@ -111,12 +112,28 @@ export class PDS {
       config.sequencerLeaderLockId,
     )
 
-    const mailTransport =
+    const serverMailTransport =
       config.emailSmtpUrl !== undefined
         ? createTransport(config.emailSmtpUrl)
         : createTransport({ jsonTransport: true })
 
-    const mailer = new ServerMailer(mailTransport, config)
+    const moderationMailTransport =
+      config.moderationEmailUser !== undefined &&
+      config.moderationEmailPassword !== undefined
+        ? createTransport({
+            service: 'gmail',
+            auth: {
+              user: config.moderationEmailUser,
+              pass: config.moderationEmailPassword,
+            },
+          })
```

Comment:
If we don't need to use the special gmail transport here, could we configure it with an SMTP url?  This would be convenient as it's pretty flexible and consistent with the other mail transport.

Reply:
reading the docs it looks like this is the only way to use app specific password and otherwise, we would have to turn on something called `less secure apps` on the google account which isn't recommended. but I could have missed something so lmk if you know a way to use the stmp transport.

Reply:
The best information I've been able to find about it is [here](https://wpmailsmtp.com/gmail-less-secure-apps/), perhaps you have a better resource (I really struggled to find official google documentation!).

My understanding is that now you _must_ turn "less secure apps" off, and _must_ use app passwords to send SMTP email.  Luckily this is what we want to do anyway, so I think that the SMTP option should work based on that information.  Let me know what info you're seeing and what you think!

Reply:
yeah that adds up with what I've found but may be I'm misunderstanding what you're asking here? I've swapped out the `service: 'gmail'` prop with smtp host config in this commit https://github.com/bluesky-social/atproto/pull/1312/commits/0ca129669a5641153a16ca55d5f6a51e60bc8447 LMK if this isn't what you wanted. honestly, I'm not sure what's the difference between the 2 configs because I'd assume nodemailer just use the smtp config behind the scene when service is set to 'gmail'?

Reply:
Ah I'm sorry for the confusion!  The idea is just to make the implementation not tied to gmail if possible.  Then both the moderation and non-moderation mailers would have the same kind of configuration: each just needs a single SMTP url.  When we configure the PDS we'll include `smtp.gmail.com` in the SMTP url ourselves, rather than it being part of the PDS's implementation.

If we go this direction my recommendation would be to replace `moderationEmailUser` and `moderationEmailPassword` config with a single `moderationEmailSmtpUrl`.  Does that seem alright?

Reply:
ahhh I got it. thanks for explaining. made some adjustments to use a singular smtp url lmk if that works?

Reply:
Sounds good!  If you made the change I'm not seeing it yet, may just need a `push`!

Reply:
🤦🏽‍♂️

Reply:
sorry, I consistently forget to push to my forked repo haha